main.rb

```diff
@@ -22,14 +23,20 @@ def runCommand(command)
       raise stderr_str
     end
 end
-runCommand("brew tap tuist/tuist && brew install tuist")
+
+runCommand("brew tap tuist/tuist")
 options = ""
 tuist_path = env_has_key("AC_TUIST_PATH")
-project_only = env_has_key("AC_TUIST_PROJECT_ONLY") || "false"
-if tuist_path
-  options += " -p #{tuist_path}"
+tuist_version = env_has_key("AC_TUIST_VERSION")
+
+if !tuist_version
+  runCommand("brew install tuist")
```

Comment:
To avoid repeating the same code (`runCommand("brew install tuist`), 
I recommend using the following command instead of if else (as in line 34-37):

```rb

 if tuist_version
   spec_version += "@#{tuist_version}"
 end

 runCommand("brew install tuist#{spec_version}")
```